### ISSUE TITLE
Support for Crimon Skies

### ISSUE THREAD

**Issue by Cosmas47** [tobywf/mech3re#1]
Just leaving a note here to let you know, these scripts also seem to work on Zipper Interactive's (2000) Crimson Skies.

Extracts:
interp "interp.zbd" - Not certain but this appears to be UI layout data mixed with mission event data.
textures "rimage.zbd" - Image data for flight briefings
sounds "soundsh.zbd" - High quality sound effects files. All *.wav.
reader "zrdr.zbd" - Animation definitions for aircraft.

So in short summary.. these findings correspond to yours. Anxious to see you crack the model ZBDs. 
If I had to guess the gamez.zbd file is most likely terrain/environment data. 
We have those in each campaign folder plus cam_anim.zbd, mis_anim.zbd, and a rTextures#.zbd

Great work, this has been a fascinating read.
